index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>chigozie ezeilo portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <a> <div class="wrapper">
+        <header>
+            <nav>
+                 <img src="./images/Logo Placeholder.svg" alt="" class="logo">
+                </a>
+                 <button class="menu"><i class="fafa-bars"></i></button>
+             </nav>
+        </header>
+        <section class="header-container">
+            <h1>Hey There. I'm chigozie</h1>
+            <h3>I'm a software developer</h3>
```

Comment:
- [ OPTIONAL ] I recommend adding more font weight on the heading to make it looks like the Figma template as required so kindly add it in all your headings 🔥💯🚀😊💪🏻